ReactWindows/ReactNative.Shared/UIManager/DependencyObjectExtensions.cs

```diff
@@ -167,6 +167,79 @@ public static ThemedReactContext GetReactContext(this DependencyObject view)
             return elementData.Context;
         }
 
+        /// <summary>
+        /// Set the view manager for the view instance. 
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="viewManager">The view manager.</param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown if view parameter is null.
+        /// </exception>
+        public static void SetViewManager(this DependencyObject view, IViewManager viewManager)
+        {
+            if (view == null)
+                throw new ArgumentNullException(nameof(view));
+
+            s_properties.GetOrCreateValue(view).ViewManager = viewManager;
+        }
+
+        /// <summary>
+        /// Get the view manager for the view instance. 
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <returns>The  implementation of IViewManager.</returns>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if view manager is not available for the view.
+        /// </exception>
+        public static IViewManager GetViewManager(this DependencyObject view)
```

Comment:
>GetViewManager [](start = 35, length = 14)

I don't see where `GetViewManager` is used anywhere. Can we eliminate this (and `SetViewManager`) for now until there is a specific use case for it.